### ISSUE TITLE
Polling for generation status in CC and TxP causes browser scroll to reset between intervals

### ISSUE THREAD

**Issue by chrisknoll** [OHDSI/Atlas#949]
### Expected behavior
When sources are refreshed, the view should only update existing observables, not create the entire view model again (causing a quick refresh of the entire DOM, leading to the reset of the scroll)

### Actual behavior
Due to the way the view-model in the templates (or possibly how the execution groups are being constructed, not sure yet), the entire dom element containing the groups are being re-built between refreshes:

```
<!-- ko foreach: executionGroups -->
<div data-bind="template: {
		name: 'pathway-view-edit-execution-group',
		data: Object.assign({}, $data, 
		{
			classes: $parent.classes,
			execColumns: $parent.execColumns,
			isExpanded: $parent.expandedSection() === $index(),
			toggleSection: $parent.toggleSection.bind(null, $index())
		})
}"></div>
<!-- /ko -->
```

The actual, correct behavior should establish the execution Groups one time, and between refreshes, push/delete any executionGroup that is added/removed, and iwhtin the execution group, only update those executions that have changes (such as we find out that another client has started a new execution under one of the groups.  Since we have executionIDs that are unique across all generations, we should be able to easily find out that an exeution has been added, and using a value compare, we can understand if a speific exeuction has changed.

### Steps to reproduce behavior
1. Go to the execution results of a CC.
2. Open Chrome dev tools, select one of the dom elements for a given group.
3. Wait for the interval, you'll notice that the selected dom element is destroyed and a new one replaced

**Comment by anthonysena** [comment]
I believe this was closed via #956 since this branch was merged and the PR closed. Please re-open if I have this mistaken. Thanks!

**Comment by chrisknoll** [comment]
I addressed this, sorry for not marking it complete.
